remote/remote_state_s3.go

```diff
@@ -14,11 +15,12 @@ import (
 
 // A representation of the configuration options available for S3 remote state
 type RemoteStateConfigS3 struct {
-	Encrypt string
-	Bucket  string
-	Key     string
-	Region  string
-	Profile string
+	Encrypt   string `mapstructure:"encrypt"`
+	Bucket    string `mapstructure:"bucket"`
+	Key       string `mapstructure:"key"`
+	Region    string `mapstructure:"region"`
+	Profile   string `mapstructure:"profile"`
+	LockTable string `mapstructure:"lock_table"`
```

Comment:
Just observing that it's a little weird that `LockTable` is part of the `RemoteStateConfigS3` struct (versus something like a `RemoteStateConfigAWS` struct, but still makes sense.